tests/simplex/simplex_project.output

```diff
@@ -0,0 +1,4521 @@
+DEAL::  line 2: 4, 3: orientation = 0
+DEAL::face 3: 0, 2, 3: orientation = 4
+DEAL::  line 0: 2, 0: orientation = 1
+DEAL::  line 1: 2, 3: orientation = 0
+DEAL::  line 2: 0, 3: orientation = 0
+DEAL::DoF 20
+DEAL::  support point -1.00000 0.00000 0.00000
+DEAL::DoF 2
+DEAL::  support point 0.00000 0.00000 0.00000
+DEAL::DoF 1
+DEAL::  support point 0.00000 1.00000 0.00000
+DEAL::DoF 3
+DEAL::  support point 0.00000 0.00000 1.00000
+DEAL::DoF 21
+DEAL::  support point -0.666667 0.00000 0.00000
+DEAL::DoF 22
+DEAL::  support point -0.333333 0.00000 0.00000
+DEAL::DoF 7
+DEAL::  support point -5.55112e-17 0.333333 0.00000
+DEAL::DoF 6
+DEAL::  support point -1.11022e-16 0.666667 0.00000
+DEAL::DoF 23
+DEAL::  support point -0.666667 0.333333 0.00000
+DEAL::DoF 24
+DEAL::  support point -0.333333 0.666667 0.00000
+DEAL::DoF 25
+DEAL::  support point -0.666667 0.00000 0.333333
+DEAL::DoF 26
+DEAL::  support point -0.333333 0.00000 0.666667
+DEAL::DoF 14
+DEAL::  support point -5.55112e-17 0.00000 0.333333
+DEAL::DoF 15
+DEAL::  support point -1.11022e-16 0.00000 0.666667
+DEAL::DoF 12
+DEAL::  support point -5.55112e-17 0.666667 0.333333
+DEAL::DoF 13
+DEAL::  support point -1.11022e-16 0.333333 0.666667
+DEAL::DoF 27
+DEAL::  support point -0.333333 0.333333 0.00000
+DEAL::DoF 28
+DEAL::  support point -0.333333 0.00000 0.333333
+DEAL::DoF 29
+DEAL::  support point -0.333333 0.333333 0.333333
+DEAL::DoF 19
+DEAL::  support point -1.11022e-16 0.333333 0.333333
+DEAL::
+DEAL::
+DEAL::
+DEAL::
+DEAL::
+DEAL::Number of cells = 2
+DEAL::
+DEAL::
+DEAL::  line 0
+DEAL::      vertex 1 = 1.00000 0.00000 0.00000
+DEAL::      vertex 3 = 0.00000 0.00000 1.00000
+DEAL::      orientation = 1
+DEAL::  line 1
+DEAL::      vertex 3 = 0.00000 0.00000 1.00000
+DEAL::      vertex 2 = 0.00000 1.00000 0.00000
+DEAL::      orientation = 1
+DEAL::  line 2
+DEAL::      vertex 2 = 0.00000 1.00000 0.00000
+DEAL::      vertex 1 = 1.00000 0.00000 0.00000
+DEAL::      orientation = 0
+DEAL::  line 3
+DEAL::      vertex 1 = 1.00000 0.00000 0.00000
+DEAL::      vertex 0 = 0.00000 0.00000 0.00000
+DEAL::      orientation = 1
+DEAL::  line 4
+DEAL::      vertex 3 = 0.00000 0.00000 1.00000
+DEAL::      vertex 0 = 0.00000 0.00000 0.00000
+DEAL::      orientation = 1
+DEAL::  line 5
+DEAL::      vertex 2 = 0.00000 1.00000 0.00000
+DEAL::      vertex 0 = 0.00000 0.00000 0.00000
+DEAL::      orientation = 1
+DEAL::vertices =
+DEAL::  1: 1.00000 0.00000 0.00000
+DEAL::  3: 0.00000 0.00000 1.00000
+DEAL::  2: 0.00000 1.00000 0.00000
+DEAL::  0: 0.00000 0.00000 0.00000
+DEAL::face 0: 1, 3, 2: orientation = 1
+DEAL::  line 0: 1, 3: orientation = 1
+DEAL::  line 1: 3, 2: orientation = 1
+DEAL::  line 2: 2, 1: orientation = 0
+DEAL::face 1: 3, 1, 0: orientation = 1
+DEAL::  line 0: 1, 3: orientation = 1
+DEAL::  line 1: 1, 0: orientation = 1
+DEAL::  line 2: 3, 0: orientation = 0
+DEAL::face 2: 1, 2, 0: orientation = 1
+DEAL::  line 0: 2, 1: orientation = 1
```

Comment:
I think I've narrowed it down to this case (two tetrahedra in weird orientations): according to the first face, the `(2, 1)` line is flipped, but according to the third face, it is not flipped. Hence - based on which face we look up information on we get different orientations, which mess up line DoFs.